samples/nrf_desktop/src/modules/usb_state.c

```diff
@@ -28,7 +29,10 @@ static enum usb_state state;
 static u8_t hid_protocol = HID_PROTOCOL_REPORT;
 static struct device *usb_dev;
 
+static struct config_channel_frame frame;
```

Comment:
`frame` is both global and local variable which may be confusing and bug prone

Reply:
done. Actually, it was causing a bug, now fixed